docs/architecture/ConnectToRedux.md

```diff
@@ -0,0 +1,121 @@
+## Connect Stateful Client to Redux
+
+Redux is a popular state management framework, it provides a centralized app state. You can easily integrate redux with all popular UI frameworks(React, Angular, VueJs). Developers can eaily integrate our stateful client with redux, and connect redux to their own exisiting framework. 
+
+![Connect with redux](../images/stateful-redux-ui.png)
+
+Here is a quick tutorial for how to connect stateful client state to redux store:
+### Install dependencies
+
+```
+npm install @azure/communication-react redux
+```
+
+### Create a reducer
+
+This reducer of redux help you update the centralized redux store with our client state.
```

Comment:
This reducer of redux help you **updates** the centralized redux store with our client state.

Reply:
This Redux reducer lets you update the centralized redux store with our client state.